src/plugin.js

```diff
@@ -182,36 +185,42 @@ const initPlugin = function(player, options) {
       details += `<span class="vjs-errors-flashmessage">${flashMessage}</span>`;
     }
     display = player.getChild('errorDisplay');
-    // The code snippet below is to make sure we dispose any child closeButtons before
-    // making the display closeable
-    if (display.getChild('closeButton')) {
-      display.removeChild('closeButton');
-    }
-    // Make the error display closeable, and we should get a close button
-    display.closeable(true);
+
     content.className = 'vjs-errors-dialog';
     content.id = 'vjs-errors-dialog';
     content.innerHTML =
```

Comment:
what do you think about building up the innerHTML string in a separate variable and when it's done being changed then setting it on `content` afterwards? It could help with performance and visual thrashing since the DOM will only be updated once.